config/environments/staging.rb

```diff
@@ -74,7 +72,4 @@
 
   # Use default logging formatter so that PID and timestamp are not suppressed.
   config.log_formatter = ::Logger::Formatter.new
-
-  # Do not dump schema after migrations.
-  config.active_record.dump_schema_after_migration = false
```

Comment:
@neerajdotname   The `db:migrate` rake task runs `dump` at the end which dumps the database structure to `schema.rb` file. It is useful in dev env, but not required in staging or production env, right ? By default, it's value is set to true.

![Screen Shot 2020-03-15 at 12 01 49 AM](https://user-images.githubusercontent.com/1580260/76688153-57856c00-6650-11ea-81fb-4da99adb3cc3.png)

It is not harm to run the task here even though it is not required. Any other specific reason to remove this flag ?
Correct me if I miss anything here.

References:
https://edgeguides.rubyonrails.org/configuring.html
https://groups.google.com/forum/#!topic/rubyonrails-core/h4cQXmKuB7M
https://github.com/rails/rails/pull/13948